src/datasources/postgres/database.ts

```diff
@@ -11,7 +11,6 @@ const db = Knex({
     user: process.env.DATABASE_USER,
     password: process.env.DATABASE_PASSWORD,
     database: process.env.DATABASE_DATABASE,
-    timezone: 'UTC',
```

Comment:
@bolmsten, do you have any opinion on this? By removing this @martin-trajanovski has made both `core` and `scheduler` backends consistent.